pyaedt/filtersolutions_core/lumped_nodes_and_leads.py

```diff
@@ -0,0 +1,220 @@
+        str
+        """
+        c_node_capacitor = self._dll_interface.get_string(self._dll.getLumpedCNodeCapacitor)
+        return c_node_capacitor
+
+    @c_node_capacitor.setter
+    def c_node_capacitor(self, c_node_capacitor):
+        self._dll_interface.set_string(self._dll.setLumpedCNodeCapacitor, c_node_capacitor)
+
+    @property
+    def c_lead_inductor(self) -> str:
+        """Series inductance assigned to each capacitor lead
+        The default is ``0``.
+
+        Returns
+        -------
+        str
+        """
+        c_lead_inductor = self._dll_interface.get_string(self._dll.getLumpedCLeadInductor)
+        return c_lead_inductor
+
+    @c_lead_inductor.setter
+    def c_lead_inductor(self, c_lead_inductor):
+        self._dll_interface.set_string(self._dll.setLumpedCLeadInductor, c_lead_inductor)
+
+    @property
+    def l_node_capacitor(self) -> str:
+        """Shunt capacitance assigned to each inductor node
+        The default is` ``0``.
+
+        Returns
+        -------
+        str
+        """
+        l_node_capacitor = self._dll_interface.get_string(self._dll.getLumpedLNodeCapacitor)
+        return l_node_capacitor
+
+    @l_node_capacitor.setter
+    def l_node_capacitor(self, l_node_capacitor):
+        self._dll_interface.set_string(self._dll.setLumpedLNodeCapacitor, l_node_capacitor)
+
+    @property
+    def l_lead_inductor(self) -> str:
+        """Series inductance assigned to each inductor lead
+        The default is ``0``.
+
+        Returns
+        -------
+        str
+        """
+        l_lead_inductor = self._dll_interface.get_string(self._dll.getLumpedLLeadInductor)
+        return l_lead_inductor
+
+    @l_lead_inductor.setter
+    def l_lead_inductor(self, l_lead_inductor):
+        self._dll_interface.set_string(self._dll.setLumpedLLeadInductor, l_lead_inductor)
+
+    @property
+    def r_node_capacitor(self) -> str:
+        """Shunt capacitance assigned to each resistor node
+        The default is ``0``.
+
+        Returns
+        -------
+        str
+        """
+        r_node_capacitor = self._dll_interface.get_string(self._dll.getLumpedRNodeCapacitor)
+        return r_node_capacitor
+
+    @r_node_capacitor.setter
+    def r_node_capacitor(self, r_node_capacitor):
+        self._dll_interface.set_string(self._dll.setLumpedRNodeCapacitor, r_node_capacitor)
+
+    @property
+    def r_lead_inductor(self) -> str:
+        """Series inductance assigned to each resistor lead
+        The default is ``0``.
+
+        Returns
+        -------
+        str
+        """
+        r_lead_inductor = self._dll_interface.get_string(self._dll.getLumpedRLeadInductor)
+        return r_lead_inductor
+
+    @r_lead_inductor.setter
+    def r_lead_inductor(self, r_lead_inductor):
+        self._dll_interface.set_string(self._dll.setLumpedRLeadInductor, r_lead_inductor)
+
+    @property
+    def c_node_compensate(self) -> bool:
+        """Flag indicating if the capacitor values may be adjusted
```

Comment:
This description doesn't read correctly. Use "might" rather than may if needed in the rewrite.